src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+    // if (typeof data[key] === 'object') {
+
+    if (Object.keys(data[key]).length !== 0) {
```

Comment:
```suggestion
    if (Object.keys(data[key]).length) {
```

Reply:
if i write without '!==0' or // if (typeof data[key] === 'object'). i get mistake on the screen below.

Reply:
![error](https://user-images.githubusercontent.com/117648418/218119743-d5c11728-5434-4b46-babd-e0af9522b55d.png)

Reply:
the problem is not with the code, but with the node version or something else
feel free to ask about it in the slack chat